helm/values-notprod.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+hocs-generic-service:
 
-deploymentAnnotations:
-  downscaler/downtime: "Mon-Sun 00:00-07:55 Europe/London,Mon-Sun 18:05-24:00 Europe/London"
+  deployment:
+    annotations:
+      downscaler/downtime: "Mon-Sun 00:00-07:55 Europe/London,Mon-Sun 18:05-24:00 Europe/London"
```

Comment:
I still feel these hours are inappropriate - I for one work compressed hours so these services should be runnable with AWS resources between 7-5 or 9-7 for me. 

There was reasoning as we shouldn't be working on these 'out-of-hours' but that also doesn't mean that tooling should be unable to run then...